content/post/cars-and-can.md

```diff
@@ -0,0 +1,159 @@
++++
+date = "2017-10-21T13:37:00+01:00"
+description = "Use a cars CAN bus to read live data."
+title = "TED2: Cars and CAN"
+series = ["Ted"]
+slug = "cars-and-can"
+author = "Auke Willem Oosterhoff"
++++
+
+In [previous post]({{< relref "post/obd_ii_elm327_and_python.md">}}) of this
+series I demonstrated how to obtain live data of a car by using an OBD-II
+adapter. I was a bit disappointend about the fact that I could only read a
+handfull of parameters. Together with Wisse Hooghiem I went a step further by
```

Comment:
- `handfull -> `handful`
-  It's nice to refer to Wisse using his Github profile, so people are getting a slightly better idea who he is :p.

Reply:
Well, I asked him this. But he told me that his name was enough.